tests/python/unittest/test_symbol.py

```diff
@@ -115,7 +115,7 @@ def test_symbol_infer_shape_var():
     shape = (2, 3)
     a = mx.symbol.Variable('a', shape=shape)
     b = mx.symbol.Variable('b')
-    c = a+b
+    c = mx.symbol.ewise_plus(a, b)
```

Comment:
again, that is a feature relies on backward inference, which strictly should not occur until NNVM.

Since operator+ already defaults to broadcast_add in the nd array side, it is nice to be consistent